core/app/models/mno_enterprise/dashboard.rb

```diff
@@ -53,7 +53,7 @@ def filtered_widgets_templates
     end
 
     def sorted_widgets
-      ids = self.widgets_order | self.widgets.map(&:id)
+      ids = self.widgets_order.map(&:to_s) | self.widgets.map(&:id)
```

Comment:
Will this still work with API v1 ? Won't `self.widgets.map(&:id)` return integers and `self.widgets_order.map(&:to_s)` strings?